tests/ansible/integration/action/synchronize.yml

```diff
@@ -40,19 +40,27 @@
     #     state: absent
     #   become: true
 
-    - synchronize:
-        private_key: /tmp/synchronize-action-key
-        dest: /tmp/sync-test.out
-        src: /tmp/sync-test/
+    # exception:   File "/tmp/venv/lib/python2.7/site-packages/ansible/plugins/action/__init__.py", line 129, in cleanup
+    # exception:     self._remove_tmp_path(self._connection._shell.tmpdir)
+    # exception: AttributeError: 'get_with_context_result' object has no attribute '_shell'
+    # TODO: looks like a bug on Ansible's end with 2.10? Maybe 2.10.1 will fix it
+    # https://github.com/dw/mitogen/issues/746
+    # - name: do synchronize test
+    #   block:
+    #     - synchronize:
+    #         private_key: /tmp/synchronize-action-key
+    #         dest: /tmp/sync-test.out
+    #         src: /tmp/sync-test/
```

Comment:
Suggestion: Skip with a `when` clause, rather than commenting them out. That way they're still visible in test output, and so less likely to be forgotten.

Reply:
👍 good point!